go/vt/key/key_test.go

```diff
@@ -633,9 +634,9 @@ func TestGenerateShardRanges(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			got, err := GenerateShardRanges(tt.args.shards)
 			if tt.wantErr {
-				assert.Error(t, err)
+				require.Error(t, err)
 			} else {
```

Comment:
you no longer need the `else` here (this is the benefit of `require`)

this should read:

```go
if tt.wantErr {
    assert.Error(t, err)
    return
}

require.NoError(t, err)
assert.Equal(t, got, tt.want)
```

Reply:
Fair but I've blindly copy pasted what was there before:
https://github.com/vitessio/vitess/blob/7c532a533d40d049d81c4cc9676ff921064e5700/go/vt/vtctl/vtctl_test.go#L59-L70

Reply:
🤦 haha sorry, and thank you for cleaning up my original mistake